packages/react/rollup.config.js

```diff
@@ -77,11 +79,11 @@ const getConfig = (format, extractCSS) => ({
     resolve(),
     ts(),
     format === 'esm' &&
-      babel({
-        babelHelpers: 'runtime',
-        exclude: 'node_modules/**',
-        extensions,
-      }),
+    babel({
+      babelHelpers: 'runtime',
+      exclude: 'node_modules/**',
+      extensions,
+    }),
```

Comment:
Prettier complains about this change. Did you make this change manually?

Reply:
Sort of, I typically format the document manually after I've made changes / on save. I'm using Prettier myself so this is a interesting phenomenon 👀

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1011/commits/5241eb696abd784318f96bcc96b14d1a932163ec